src/os/ui/layer/vectorlayerui.js

```diff
@@ -977,3 +998,34 @@ os.ui.layer.VectorLayerUICtrl.prototype.onRotationColumnChange = function(event,
     this.createCommand(fn);
   }
 };
+
+
+/**
+ * Handle map property changes.
+ * @param {os.events.PropertyChangeEvent} event The change event.
+ * @private
+ */
+os.ui.layer.VectorLayerUICtrl.prototype.onMapView3DChange_ = function(event) {
+  if (event.getProperty() == os.MapChange.VIEW3D) {
+    this['showAltitudeModes'] = event.getNewValue() && this['altitudeModes'].length > 0;
```

Comment:
Should i change this to > 1. As it stands the drop down menu can appear with only one option

Reply:
Selects with a single option aren't great, but hiding it may be alarming to a user that's used to the control in Cesium (if they switch to Strata and Strata doesn't have full support yet). I think this is fine as is, and I would also expect Strata to have this support before we release it.